chia/util/initial-config.yaml

```diff
@@ -242,6 +242,10 @@ timelord:
   # You must set 'send_uncompact_interval' in 'full_node' > 0 in the full_node
   # section below to have full_node send existing time proofs to be sanitized.
   sanitizer_mode: False
+  # This runs a less CPU intensive bluebox. Runs for windows. Settings apply as for `sanizer_mode`.
```

Comment:
"sanitizer_mode"

Reply:
I ended up renaming `sanitizer_mode` into `bluebox_mode` (but it should keep the backwards compatibility so it works even without redoing `chia init`).